pyof/v0x01/common/action.py

```diff
@@ -196,7 +197,7 @@ class ActionVlanVid(ActionHeader):
     _allowed_types = ActionType.OFPAT_SET_VLAN_VID,
 
     def __init__(self, vlan_id=None):
-        """The following constructor parameters are optional.
+        """Create a ActionVlanVid with the optional parameters below.
```

Comment:
a**n** (I'll stop here, but there's lots of them).

Reply:
there are*
:p